src/app/components/Figure/index.jsx

```diff
@@ -1,9 +1,13 @@
 import React from 'react';
 import { string } from 'prop-types';
 import styled from 'styled-components';
+import Copyright from './Copyright';
```

Comment:
Does this need to be it's own component? Feels a bit over-kill. Could it not just render the copyright into `VisuallyHiddenText`?

Reply:
We could. However this way if we need to extend Copyright to be visual (not just offscreen) it doesn't need to be pulled out of Figure, also it means in the future the Figure presentational component doesn't need to worry about the translation for the offscreen text `Copyright `. 

This is likely mitigated if we have a Figure container component which I have suggested following this PR to implement the Copyright offscreen text.

Reply:
I think personally the best solution would be to have a Figure container which handles the setting/translation of the offscreen text (`Copyright `) and then renders the VisuallyHiddenText therefore removing the need for the Copyright component and also not putting the offscreen text logic in a presentational component. That said I think that solution would be out of scope for the current ticket and should follow this PR